examples/data-sources/nios_dhcp_dhcpfailover/data-source.tf

```
@@ -0,0 +1,18 @@
+// Retrieve a DHCP failover by filter using name
```

```suggestion
// Retrieve a specific DHCP failover using filters
```
Nitpick
Maintaining convention